Tasks/AzureFileCopyV2/package.json

```diff
@@ -4,7 +4,7 @@
     "@types/mocha": "2.2.48",
     "@types/node": "6.0.68",
     "@types/q": "1.0.7",
-    "azure-arm-rest-v2": "file:../../_build/Tasks/Common/azure-arm-rest-v2-2.0.0.tgz",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.2",
```

Comment:
This is 1.0.2 because the original published version had some TS files in it. Its not a big deal and wouldn't have functional impact, but was causing a compile error on one of the tasks.